lab-ron/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
This is almost perfect, really good work.  "gulp" and "gulp-eslint" should be in devDependencies not dependencies.  This is because they are not necessary for the program to run (only needed during development).